app/views/content/index.html.erb

```diff
@@ -11,8 +11,10 @@
       </div>
 
 
+
       <div class="filter-form">
         <%= form_tag content_path, method: 'get', name: 'organisation-picker', id: 'filter-form' do %>
+          <input type="hidden" name="submitted" value="true" >
```

Comment:
This seems a bit flaky, but I do get the need to separate `All` from the other options.